modules/web-console/web-agent/src/main/java/org/apache/ignite/console/agent/db/dialect/MySQLMetadataDialect.java

```diff
@@ -39,12 +39,13 @@
     }
 
     /** {@inheritDoc} */
-    @Override public Collection<String> schemas(Connection conn) throws SQLException {
+    @Override public Collection<String> schemas(Connection conn, boolean importSamples) throws SQLException {
         Collection<String> schemas = new ArrayList<>();
 
         ResultSet rs = conn.getMetaData().getCatalogs();
```

Comment:
The only difference between  MySQLMetadataDialect  and JdbcMetadataDialect is this line:
JdbcMetadataDialect: ResultSet rs = conn.getMetaData().getSchemas();
MySQLMetadataDialect : ResultSet rs = conn.getMetaData().getCatalogs();

Almost the same for OracleMetadataDialect.

May be it make sense to refactor as method that will return collection strings with schemas names?

Reply:
Removed code duplication on collection of schemas.